eng/Version.Details.xml

```diff
@@ -58,6 +58,138 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>e1fa5d7648d46f067e265211fc2c695d409fe788</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.Extensions.Caching.Abstractions" Version="5.0.0-preview.3-runtime.20181.3" CoherentParentDependency="Microsoft.NETCore.App.Runtime.win-x64">
```

Comment:
Please remove all CPD attributes. They aren't needed for packages coming from a single repo (and we don't need to link arcade or roslyn dependencies to the runtime repo).

Reply:
Hmm I suppose that's my bad for following the format of the other System.* dependencies.